service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/selector/ChildSelector.java

```diff
@@ -15,26 +12,47 @@
 
 @Data
 @Builder
-public class ChildSelector {
-    @Setter(AccessLevel.PRIVATE)
+public class ChildSelector implements Selector {
```

Comment:
What's the implementationo difference between CareOrderSelector and this one? Can't it be further simplified?

Reply:
The name of property. Changed to have single representation